assets/style/main.css

```diff
@@ -0,0 +1,517 @@
+  background-color: #fff;
+}
+
+.header-nav {
+  min-width: 375px;
+  padding: 13px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mobnavbar-link {
+  display: none;
+}
+
+.navbar-logo {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-left: 40px;
+}
+
+.logo-link {
+  font-weight: 700;
+  font-size: 18px;
+  margin: 0 20px;
+  padding: 10px;
+}
+
+.navbar-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: 25px;
+}
+
+.navbar-link {
+  font-weight: 500;
+  font-size: 15px;
+  padding: 10px;
+  margin: 0 20px;
+}
+
+.grid-container {
+  min-width: 375px;
+  background-color: #f7f7f9;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: auto;
+  padding-top: 50px;
+  margin-bottom: 0;
+}
+
+.grid-item {
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.grid-item-1 {
+  min-height: 95vh;
+  background-image: url("../images/desktop/introduction.png");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 15%;
+  margin-bottom: 40px;
+}
+
+.intro-container {
+  margin: 0 20px;
+  width: 70%;
+}
+
+.link-img {
+  height: 20px;
+  width: 20px;
+  margin: 10px;
+}
+
+.list-social {
+  display: flex;
+  align-items: flex-end;
+}
+
+.work-container {
+  display: flex;
+  align-items: stretch;
+  border-radius: 20px;
+  margin: 60px;
+}
+
+.mlr-20 {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.mall-30 {
+  margin: 30px;
+}
+
+.section-img {
+  width: 50%;
+  border-radius: 8px;
+}
+
+.section-content {
+  width: 50%;
+  border-radius: 8px;
+}
+
+.img-content {
+  width: 100%;
+  height: 100%;
+}
+
+.flexr-default {
+  display: flex;
+  justify-items: center;
+}
+
+.mt-30 {
+  margin-top: 30px;
+}
+
+.work-link {
+  margin: 1px;
+  padding: 10px;
+}
+
+.flex-list {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.dot-icon {
+  width: 10px;
+  height: 10px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.lang-link {
+  margin: 2px 5px;
+  padding: 8px;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.btn-more {
+  margin: 10px;
+  padding: 12px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+}
+
+.btn-more:hover {
+  color: #fff;
+  border: 1px solid #7f8cff;
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+.btn-more:disabled {
+  background-color: #fff;
+  border: 1px solid #5e6c84;
+  color: #5e6c84;
+}
+
+.btn-more:active {
+  background: #2230d2;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.grid-item-3 {
+  border-top-right-radius: 50px;
+  margin: 30px;
+}
+
+.about-container {
+  display: flex;
+  align-items: flex-start;
+  margin: 30px;
+}
+
+.about-myself {
+  width: 45%;
+  margin: 30px;
+}
```

Comment:
- [ ] According to the given template, every `text` item inside the card should be left aligned. But in your card, `LET'S CONNECT` and `SEE PROJECT` buttons are not left aligned. Kindly fix this.
- [ ] There should not be any space between `My and self`. Kindly fix this issue also.

![image](https://user-images.githubusercontent.com/60083437/221145612-c88e61a6-8555-45b5-bc70-0cd2dc0b0e37.png)